### PR TITLE
Work out deserialization from db #6

### LINKED ISSUE #6: work out deserialization from db

### LINKED ISSUE #9: add bootstrap

### PR DESCRIPTION
**Pull request by illuminatedspace**
closes #6  closes #9

### CODE REVIEW

File: app/components/Render.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          className='element-display'
+          style={{left: `${ele.x}px`, bottom: `${ele.y}px`}}
+        />
+      )
+    }
+  }
+
+  render () {
+    console.log("PROPS", this.props)
+    console.log("KEYS", Object.keys(this.props.page))
```
Comment (luisamiranda) [inline app/components/Render.jsx]: do we want to commit these console.logs to master?

File: app/components/Render.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    const keys = Object.keys(this.props.page)
+    return (
+      <div>
+        <h1>HIZZZ</h1>
```
Comment (luisamiranda) [inline app/components/Render.jsx]: What is HIZZZ? It sounds great.

File: app/main.jsx [hunk 3]
```diff
@@ -27,8 +38,9 @@ render (
       <Route path="/" component={ExampleApp}>
         <IndexRedirect to="/jokes" />
```
Comment (luisamiranda) [inline app/main.jsx:39]: do we want to leave /jokes as the homepage?

File: public/index.html [hunk 4]
```diff
@@ -1,9 +1,11 @@
 <!doctype html>
 <html>
   <head>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
```
Comment (luisamiranda) [inline public/index.html]: we made a change to index.html too. we added the CDN for jquery, which bootstrap also needs. maybe add it here and we can discard our change

**Review by ariamoraine — APPROVED** [review]

**Pull request merged**